softdevice_controller/doc/bluetooth_coex_example_diagrams.rst

```diff
@@ -1,13 +1,9 @@
 .. _bluetooth_coex_example_diagrams:
 
-Bluetooth External Radio Coexistence examples
+Bluetooth External Radio Coexistence Examples
 #############################################
 
-.. contents::
-   :local:
-   :depth: 2
-
-3-Wire interface
+3-Wire Interface
```

Comment:
This needs to be reverted. I was included in https://github.com/nrfconnect/sdk-nrfxlib/commit/a7b2fd1fdc65417c6035c645b02c3064f159d88b